scenarios/openshift/ibmcloud_node_scenarios.yml

```diff
@@ -0,0 +1,9 @@
+# yaml-language-server: $schema=../plugin.schema.json
+- id: ibmcloud-node-reboot
```

Comment:
We might want to add in all the available options: 
id: <ibmcloud-node-terminate/ibmcloud-node-reboot/ibmcloud-node-stop/ibmcloud-node-start>. Thoughts?